frontend/components/puzzle2.jsx

```diff
@@ -1,20 +1,46 @@
 import React from 'react';
+import { Route, Redirect } from 'react-router-dom';
 
 // Answer: See o double you
-
 class Puzzle2 extends React.Component {
-  render() {
 
-    return (
-      <div>
+ constructor(props) {
+    super(props);
+    
+    this.state = {
+      puzzlesolved: 'no',
+      blankanswer: 'no',
+      wronganswer: 'no'
+    }
+ };
+   submitanswer() {
+      var userinput = document.getElementById('puzzleAnswer').value;
+      if (userinput) {
+      this.props.send('submit', {puzzle: '1', answer: userinput})
+      console.log('answer was submitted')
+  } else {
+    this.setState({blankanswer : 'yes'})
```

Comment:
Not convinced we need to do something different for a blank answer - I think we should still submit them, but they will probably be wrong answers

Reply:
Okay. Will update to still send an empty response.